packages/editor-ui/tsconfig.json

```diff
@@ -10,11 +10,8 @@
 		"importHelpers": true,
 		"allowSyntheticDefaultImports": true,
 		"baseUrl": ".",
-		"typeRoots": [
```

Comment:
are these even needed anymore?
I'm able to remove `typeRoots` and `types`, and still able to build and test everything in this package.